CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 1.19.1 (Unreleased)
```

Comment:
There is no 1.19.1 for tpgb, though? I guess I don't know why we're updating the CHANGELOG at all, here.

Reply:
Ignore me, I can't read.